packages/design-system/src/components/OcTag/OcTag.vue

```diff
@@ -113,4 +126,9 @@ function $_ocTag_click(event: MouseEvent) {
     }
   }
 }
+
+.oc-tag-color-custom-purple {
+  background-color: #ca8df5;
+  color: white;
```

Comment:
`background-color` would then e.g. be `var(--oc-role-primary)` and `color` would be `var(--oc-role-on-primary)`.